homeassistant/components/logbook/__init__.py

```diff
@@ -490,16 +508,17 @@ def yield_rows(query: Query) -> Generator[Row, None, None]:
             # so we don't switch over until we request > 1 day+ of data.
             #
             rows = query.yield_per(1024)
+
         for row in rows:
             context_lookup.setdefault(row.context_id, row)
-            if row.context_only:
-                continue
-            event_type = row.event_type
-            if event_type != EVENT_CALL_SERVICE and (
-                event_type == EVENT_STATE_CHANGED
-                or _keep_row(hass, event_type, row, entities_filter)
-            ):
-                yield row
+            if not row.context_only:
+                event_type = row.event_type
+                if event_type != EVENT_CALL_SERVICE and (
+                    entities_filter is None
```

Comment:
We might need to still enter the function if its a `context_id` or `entity_id` select to make sure we don't show stop/start events when selecting these.

Reply:
Doesn't appear so as they get filtered in the SQL.